packages/components/src/Button/button.stories.tsx

```diff
@@ -16,7 +15,10 @@ export default {
   },
 };
 
-type Args = React.ComponentProps<typeof Button>;
+type Args = React.ComponentProps<typeof Button> & {
+  "data-testid"?: string;
```

Comment:
One of the stories is providing a `data-testid`

Reply:
should we add this to the Button component itself (will this be an issue for consumers of Button that try to use `data-testid`)?

Reply:
We definitely could, and probably should.

Here I just directly fixed the type failure (in the story), but will switch it over to the component.

Reply:
Done